ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -15,6 +15,201 @@
 from mlagents.trainers.torch.distributions import DistInstance, DiscreteDistInstance
 
 
+class AgentAction(NamedTuple):
```

Comment:
I know this comes a bit late, but since this class is pretty large, maybe we should have a `agent_action.py` for it?

Reply:
How about `action_utils.py` and we can include `ActionLogProbs` as well. Or would you prefer each have their own file?

Reply:
I would prefer each their own files